front/src/routes/integration/style.css

```
@@ -0,0 +1,190 @@
+  border-radius: 4px;
+  display: block;
+  flex-shrink: 0;
+}
+
+.integrationListDescription {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  margin-top: 2px;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)